packages/postgres-database/tests/test_models_tags.py

```diff
@@ -0,0 +1,47 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+
+
+import sqlalchemy as sa
+from simcore_postgres_database.models.base import metadata
+from simcore_postgres_database.models.tags import tags_to_groups
+from simcore_postgres_database.models.users import users
+
+
+def test_migration_downgrade_script():
+    # This test keeps how the downgrade expression in
+    # migration/versions/3aa309471ff8_add_tag_to_groups_table_and_rm_tag_user_.py
```

Comment:
Q: I am wondering whether it makes sense to run this test with each job? the migration will never change. I do get the point when you develop it but once it's there and working it will never change right? Maybe this should go with a special test suite?